frontend/src/screens/wallet/swap/SwapOutStatus.tsx

```diff
@@ -130,7 +130,7 @@ export default function SwapOutStatus() {
                       <div className="flex items-center text-muted-foreground text-sm">
                         <Loading className="w-5 h-5 mr-2" />
                         <div className="flex items-center gap-2">
-                          <p>Waiting for onchain confirmation...</p>
+                          <p>Waiting for 2 onchain confirmations...</p>
```

Comment:
Whoops, this should be 2/1 based on whether claimTxId is available I'll change that

Reply:
Thanks!